src/styles/blocks/cards.scss

```
@@ -0,0 +1,119 @@
+body {
+  margin: 0 ;
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+}
+
+p,
+h2 {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+  }
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: #060b35;
+  }
+  &__text {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: #616070;
+    font-style: normal;
+    margin-bottom: 16px;
+
+  }
+  &__review-count {
+    font-size: 10px;
+    font-weight: 400;
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+  &__prices {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: #616070;
+  }
+  &__suma {
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 18px;
+  }
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
+    height: 40px;
+
+    background-color: $colorback;
+    border: 1px solid $colorback;
+    border-radius: 5px;
+
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    font-weight: 700;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    transition: 0.3s;
+  }
+
+  &__button:hover {
+    background-color: #fff;
+    color: $colorback;
+    border: 1px solid $colorback;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
    border-color: $colorback;
```

---

Still need to fix it.